istio.deps

```diff
@@ -1,5 +1,6 @@
 [
 	{
+		"_comment": "This file is auto updated by a robot executing `https://github.com/istio/test-infra/blob/master/prow/test-infra-update-deps.sh`.",
```

Comment:
:tada:

does the bot deal with vendor yet ?

Reply:
No fix for vendor yet and the bot is still disabled. krishna will be working on it although all of us are pretty occupied with other work. Still, having this comment at least explains what this file is for (and I got this question quite a few times)